### PR TITLE
Add files

### PR DESCRIPTION
**Pull request by Thraka**
This is a test of the body of the issue

### CODE REVIEW

**Comment by Thraka** [comment]
test

**Comment by Thraka** [comment]
comment

**Comment by Thraka** [comment]
ccc ade

**Comment by Thraka** [comment]
fda

**Comment by Thraka** [comment]
#please-review

File: docs/core/about.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
+---
+title: About .NET Core
+description: Learn about .NET Core.
```
Comment (SadConsole) [inline docs/core/about.md]:
```suggestion
description: Learn about .NET Core.aaaa
```

**Comment by Thraka** [comment]
test

**Comment by Thraka** [comment]
#please-review

**Comment by Thraka** [comment]
#please-review

**Comment by Thraka** [comment]
#please-review

**Comment by SadConsole** [comment]
#please-review

**Comment by Thraka** [comment]
#please-review

**Comment by Thraka** [comment]
#please-review

**Comment by Thraka** [comment]
Just leave a comment with a single line of: #please-review

**Pull request closed without merging**